.github/workflows/build.yml

```diff
@@ -118,7 +118,7 @@ jobs:
         run: cd packages/api && POSTGRES_HOST=127.0.0.1 POSTGRES_DB=postgres POSTGRES_USER=postgres POSTGRES_PASS=postgres POSTGRES_DB=postgres npm run db:migrate
 
       - name: Run integration tests
-        run: cd packages/api && POSTGRES_HOST=127.0.0.1 POSTGRES_DB=postgres POSTGRES_USER=postgres POSTGRES_PASS=postgres POSTGRES_DB=postgres npm run test:integration
+        run: cd packages/api && DAPI_HOST=127.0.0.1 DAPI_PORT=1443 DAPI_PROTOCOL=http POSTGRES_HOST=127.0.0.1 POSTGRES_DB=postgres POSTGRES_USER=postgres POSTGRES_PASS=postgres POSTGRES_DB=postgres npm run test:integration
```

Comment:
Why we need these? We are mocking everything in the tests

Reply:
we cannot mock an instance of DAPIClient or the constructor in which it is created, it requires at least some data in order not to get an assertion error when initializing it, but it is not used anywhere in the tests afterwards

Reply:
Move it into utils and mock it. Defining dumb params just because its not working without that while not really necessary looks weird

Reply:
✅